tests/browsable_api/test_browsable_nested_api.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import unicode_literals
+
+from django.conf.urls import url
+from django.test import TestCase, override_settings
```

Comment:
Looks to me like we need this:

    from django.test.utils import override_settings

To support full range of Django versions that we cover.